### ISSUE TITLE
Dota plus in game networth feature

### ISSUE THREAD

**Issue by nikthedruid** [jeffhill/Dota2#1079]
### Description

I cant see my networth anymore. I bought dota plus yesterday and it only works for a few games.

**Comment by viper9503** [comment]
Yes, I second this, I am a long-term subscriber and noticed this. 

Since the last 3-4 days the dota plus networth gets stuck after some time into the match I don't think it is precise timing. It is different for everyone, for example, my party member's dota plus got stuck at around 23 mins while mine was working then but later on mine got stuck as well. This includes LH/DN and networth.
